cmd/internal/backup/providers/s3/versions.go

```diff
@@ -1,23 +1,23 @@
 package s3
 
 import (
-	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
 	"github.com/metal-stack/backup-restore-sidecar/cmd/internal/backup/providers"
 	"github.com/metal-stack/backup-restore-sidecar/cmd/internal/backup/providers/common"
 )
 
-// backupVersionsS3 contains the list of available backup versions
-type backupVersionsS3 struct {
-	objectAttrs []*s3.ObjectVersion
+// BackupVersionsS3 contains the list of available backup versions
+type BackupVersionsS3 struct {
```

Comment:
Why exported ? I can only see package usage

Reply:
Probably just because the PR is so old and something happened during pull. Also think this should be package internal.